doc/RPC_Layer_Core.rst

```diff
@@ -60,7 +60,7 @@ Requirements
 
 - [r.rpccore.efficient.bulk] 0-copy, if provided by the underlying network transport, is utilized;
 
-- [r.rpccore.eos] support ordered exactly once semantics (EOS) of delivery;
+- [r.rpccore.cortx] support ordered exactly once semantics (CORTX) of delivery;
```

Comment:
In this document, EOS means "Exactly-once semantic". See https://datatracker.ietf.org/doc/html/rfc8881#section-2.10.6.

Reply:
I didn't notice this.
This should be reverted. Or to remove this warning, we may have:
```
[r.rpccore.exactly-once-semantics] support ordered exactly once semantics of delivery;
```